packages/camp/package.py

```diff
@@ -216,7 +216,9 @@ class Camp(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("cub", when="+cuda")
 
-    depends_on("blt")
+    depends_on("blt", type="build")
+    depends_on("blt@0.6.0:", type="build", when="@2023.06.99:")
```

Comment:
Is the '99' correct here and below?

Reply:
So the `:` follows the Python list access pattern which is inclusive. So this is the convention some people use in Spack when you want to be above a version but not include it.

I am up for alternatives if you like, or I can go back when the new version shows up and fix this.

Reply:
Thanks for the explanation. I can put up a PR in this project after the new releases come out.